src/ResourceManager/Profile/Commands.Profile/Az.Accounts.psd1

```diff
@@ -57,24 +57,24 @@ DotNetFrameworkVersion = '4.7.2'
 RequiredAssemblies = '.\Microsoft.Azure.PowerShell.Authentication.Abstractions.dll', 
     '.\Microsoft.Azure.PowerShell.Authentication.dll', 
     '.\Microsoft.Azure.PowerShell.Authentication.ResourceManager.dll', 
-    '.\Microsoft.Azure.PowerShell.Authorization.dll', 
-    '.\Microsoft.Azure.PowerShell.Compute.dll', 
-    '.\Microsoft.Azure.PowerShell.Graph.Rbac.dll', 
-    '.\Microsoft.Azure.PowerShell.Network.dll', 
-    '.\Microsoft.Azure.PowerShell.ResourceManager.dll', 
+    '.\Microsoft.Azure.PowerShell.Clients.Authorization.dll', 
```

Comment:
Just a quick question: are we going to change the profile directory as part of this, or is that too big a change?

Reply:
No directory changes/moving files happened in az-cleanup. It would have caused a lot of unnecessary merge conflicts and had no benefits for Az 1.0. I'll be cleaning/updating the repo in the next few weeks to reduce the directory structure and rename things. As a note, a lot of our processes are tied to names and directory structure. So, this would have been fairly massive to change for the entire repo right now. After Az 1.0, it will be easier to do.